docs/Data/Release_Notes/Data_Release_Notes.md

```diff
@@ -46,6 +47,122 @@
 | [v2.0](Data_Release_Notes.md#data-release-20) | August 9, 2016 |
 | [v1.0](Data_Release_Notes.md#initial-data-release-10) | June 6, 2016 |
 
+## Data Release 39.0
+
+* __GDC Product__: Data
+* __Release Date__: December 4, 2023
+
+### New Updates
+
+* New Projects
+    * MATCH-H - Genomic Characterization CS-MATCH-0007 Arm H - phs001888
+        *  21 cases
+        *  WXS, RNA-Seq
+    * MATCH-I - Genomic Characterization CS-MATCH-0007 Arm I - phs002181
+        *  60 cases
+        *  WXS, RNA-Seq
+    * MATCH-U - Genomic Characterization CS-MATCH-0007 Arm U - phs002179
+        *  23 cases
+        *  WXS, RNA-Seq
+    * MATCH-W - Genomic Characterization CS-MATCH-0007 Arm W - phs001948
+        *  45 cases
+        *  WXS, RNA-Seq
+    * MATCH-Z1A - Genomic Characterization CS-MATCH-0007 Arm Z1A - phs001973
+        *  45 cases
+        *  WXS, RNA-Seq
+
+* New Cases from Existing Projects
+    * HCMI-CMDC - 19 cases
+
+* New Data Sets
+    * 6,957 WGS alignments from the TCGA program
+    * 1,002 sets of WGS variants from TCGA
+    * MP2PRT-ALL:  WXS and RNA-Seq data
+    * Tumor-only data produced with a new pipeline. This includes raw and annotated VCFs and MAFs for the following projects. Note that all tumor-only variants are controlled-access:
```

Comment:
VAREPOP-APOLLO, MATCH-I, TARGET-ALL-P3 and TARGET-WT should be added to list, MATCH-R should be removed from list.

Reply:
Projects added/removed!